src/hb-uniscribe.h

```diff
@@ -40,6 +40,8 @@ hb_uniscribe_font_get_logfontw (hb_font_t *font);
 HB_EXTERN HFONT
 hb_uniscribe_font_get_hfont (hb_font_t *font);
 
+HB_EXTERN hb_face_t *
+hb_uniscribe_hfont_face_create (HFONT hfont);
```

Comment:
It should be `hb_gdi_*` AFAIK, Uniscribe is not involved in this API at all.

Reply:
Done